config/en.yml

```diff
@@ -6,89 +6,89 @@ en:
 
 ########################################
 ### en/account.yml
-  congratulations_you_have_signed_up: Congratulations! You have signed up for %{site_title}.
-  cookie_disabled_warning: You have cookies disabled. You will not be able to login
-    until you enable cookies.
-  email_verification_body: |-
+  congratulations_you_have_signed_up: "Congratulations! You have signed up for %{site_title}.
+  cookie_disabled_warning: You have cookies disabled. You will not be able to log in
+    until you enable cookies."
+  email_verification_body: "|-
```

Comment:
I don't think you need quotes here. The |- already is yaml syntax for a multi line value.